clirr-ignored-differences.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hard no. Please don't add methods to interfaces.

Reply:
As of now,every handwritten client follow the same behaviour if there is a new feature comes and that requireds api call,then for the rpc call we are adding it in interface,without adding methods in interface it wont be possible.help me if you have better way.